packages/support/src/settings/_variables.color.docs.scss

```diff
@@ -117,32 +117,3 @@ Markup:
 
 Style guide: style.color.status
 */
-
-
-/*
-Additional colors
```

Comment:
I'd prefer to leave this section in, if we provide the option to have alternate naming I think we should surface it somewhere. Also, if we just delete this section `muted-inverse` and `transparent` will be removed. 

Also, these "additional colors" are still surfaced on `background-color` as extra examples